content/doc/book/installing/_kubernetes.adoc

```diff
@@ -552,3 +553,183 @@ This may also be found at:
 ----
 
 You have successfully installed Jenkins on your Kubernetes cluster and can use it to create new and efficient development pipelines.
+
+==== Install Jenkins with Jenkins Operator
+
+Jenkins operator is a Kubernetes native operator who manages operations for 
```

Comment:
The Jenkins Operator is a Kubernetes native Operator which manages operations for